cms/server/AdminWebServer.py

```diff
@@ -905,48 +1242,109 @@ def get(self, manager_id):
 
 
 class AddDatasetHandler(BaseHandler):
-    """Add a dataset to a task.
+    """Add a new, clean dataset to a task.
```

Comment:
Is the equivalent to the current behavior when the dataset_id_to_copy given is equal to the string: `-`. Now AddDatasetHandler adds a new (always empty) dataset, while the new CloneDatasetHandler creates a new dataset by cloning an existing one.